src/main/java/io/cdap/e2e/pages/locators/CdfHubLocators.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * CDF hub related locators.
+ */
+public class CdfHubLocators {
+
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search by name']")
+  public static WebElement searchTab;
+  @FindBy(how = How.XPATH, using = "//input[@class='search-input form-control'][@placeholder='Search']")
+  public static WebElement searchTabControlCenter;
+  @FindBy(how = How.XPATH, using = "//div[@data-cy='hub-close-btn']")
```

Comment:
any locators with //div[@data-cy] should be used inside datacy attributes properties file with proper naming conventions . Please change this accordingly

Reply:
done.